ts/nni_manager/package.json

```diff
@@ -21,7 +21,7 @@
     "lockfile": "^1.0.4",
     "python-shell": "^2.0.1",
     "rx": "^4.1.0",
-    "sqlite3": "^5.0.0",
+    "sqlite3": "5.0.0",
```

Comment:
Use a fixed version for sqlite3? then suggest to add annotation here to explain the reason.

Reply:
Already added in description of this PR. Is there anyway to comment a JSON file?

Reply:
Ok, seems it hack for adding comments in this file.